pipeline/beam_tables.py

```diff
@@ -568,9 +568,16 @@ def run_beam_pipeline(self, scan_type: str, incremental_load: bool,
       lines = _read_scan_text(p, new_filenames)
 
       if scan_type == satellite.SCAN_TYPE_SATELLITE:
+        # For Satellite v1 - v2.1, the received IP tags (e.g. asnum) are in a
+        # separate file from results.json. Added the `received_tagging` param
+        # because the steps for adding these tags were slow in us-west2.
+        #
+        # TODO: The Satellite pipeline works with received tagging in
+        # us-central1, so this might not be necessary now.
+        received_tagging = True
```

Comment:
Since this is currently always set to True I don't think we need to pass it through. 

Since `received_tagging = True` is working I don't think we want to run pipelines with it turned off. I would remove the places where this is plumbed through, and remove the branches where `received_tagging = False` (We also don't have testing for those branches, which makes me nervous.)

That will leave us with one common behavior: always tagging received IPs
And one major change in this PR: using the existing tag data for v2.2 rows instead of caida.